packages/cli/src/sponsor/marketUtils.js

```diff
@@ -77,7 +77,7 @@ const getMarketSummary = async (web3, artifacts) => {
 };
 
 async function getCollateralizationRatio(web3, empAddress, collateral, tokens) {
-  const { toBN, toWei } = web3.utils;
```

Comment:
Wow eslint removes unused modules?? super cool

Reply:
Not automatically :(. Had to do all of these changes manually. Although the VSCode eslint plugin really helped.